index.html

```diff
@@ -52,6 +54,164 @@ <h1> Hey There. <br> I'm Dennis <br>
           </ul>
         </div>
       </section> 
+      
+      <section id="recentWorks">
+        <div class="imgHeader">
+          <h2>My Recent Works</h2>
+          <img src="./images/imgPlaceholder.png" alt="placeholderImg">
+        </div>
+        <div id="works">
+          <div class="introInfo">
+            <div class="title"><span>Multi-Post Stories</span></div>
+            <p>
+              <span>A daily selection of privately personalized reads; no accounts or sign-ups required. 
+              has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+              </span>
+            </p>
+          </div>
+          <div class="buttonGroup">
+            <div class="button">css</div>
+            <div class="button">html</div>
+            <div class="button">bootstrap</div>
+            <div class="button">Ruby</div>
+          </div>
+          <div class="bigButton">See Projects</div>
+        </div>
+        
+        <section id="seeProjects">
+          <div id="cardWork">
+            <div class="bg">
+              <div class="prjectDescription">
+                <span>Profesional Art <br/> Printing Data</span>
```

Comment:
- [ ] Kindly I recommend to you use  a header tag `hx` the project title ,This way will be semantically correct and the screen readers and other technologies will thank you. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.